src/controller/python/chip/clusters/Objects.py

```diff
@@ -27293,6 +27293,9 @@ class SimpleEnum(IntEnum):
             kValueA = 0x01
             kValueB = 0x02
             kValueC = 0x03
+            # kUnknownEnumValue will be generated the same way it is generated for .h by zap codegen
+            # and will be done for all IntEnum types
+            kUnknownEnumValue = 0x04  
```

Comment:
Please use the top of the range for all, rather than the next one (which may eventually become used)

Reply:
I plan on using the same thing that the c++ version is using. Based on the conversation over on issue https://github.com/project-chip/connectedhomeip/issues/24368 I don't think it is an issue to use the first unused here. The idea is that nothing should send out kUnknownEnumValue. Once that issue is addressed this will not be an issue

Reply:
Chatted with Tennessee about this over a call. While not what we would do if we were implementing this for this first time. We are okay with how this is done since there doesn't seems to be an issue in the for different potential cases. Assuming that https://github.com/project-chip/connectedhomeip/issues/24368 becomes fixed.